extras/kms/testing.go

```diff
@@ -116,6 +116,16 @@ func TestDb(t *testing.T) (*dbw.DB, string) {
 	return dbw.TestSetup(t, dbw.WithTestMigrationUsingDB(testMigrationFn(t)))
 }
 
+// TestDeleteKeyPurpose allows you to delete a KeyPurpose for testing.
```

Comment:
Does this delete a KeyPurpose, or a Key *with* the given KeyPurpose?

Reply:
It deletes all the keys for a given purpose.  I'll try to make that more clear.